docs/en/seccons_bcps.rst

```diff
@@ -36,7 +36,7 @@ Resolve endpoint
 ++++++++++++++++
 
 This endpoint MUST release the Metadata, the Trust Marks and the previously processed Trust Chain, and MUST
-NOT trigger a procedure of Federation Entity Discovery for each request arrival.
+NOT trigger a procedure of Federation Entity Discovery for each request arrival, unless this endpoint is secured with a client authentication mechanism, such as private_key_jwt [OIDC-CORE].
```

Comment:
questo è definito nelle considerazioni di sicurezza qui
https://openid.net/specs/openid-connect-federation-1_0.html#name-denial-of-service-attack-pr

eppoi l'endpoint può essere protetto in tanti modi, eventualmente dovremmo dire che se protetto con private_key_jwt, il metadata risolto DEVE riferirsi al subject contenuto nel private_key_jwt

altrimenti un soggetto X innescherebbe migliaia di discovery per metadata di altri soggetti, usando la sua private_key_jwt

Reply:
@fmarino-ipzs per te ok?

Reply:
Sono d'accordo con la possibilità di innescare la procedura di Discovery nel caso di autenticazione private_key_jwt ma inserendo il suggerimento di @peppelinux di risolvere solo per il sub contenuto nel private_key_jwt

Reply:
ok per me

Reply:
Mi sembra ragionevole, chiedo anche @damikael se è d'accordo.

Reply:
Anche per me è ragionevole. In ogni caso, la modifica era per ora necessaria al fine di allineare il testo alla determina di AgID. A seguire potremmo anche ragionare insieme se lasciare o meno tale possibilità.